.vscode/tasks.json

```diff
@@ -12,6 +12,21 @@
       },
       "problemMatcher": []
     },
+    {
+      "label": "Run All Linters",
```

Comment:
This label might indicate it is running against all (which it is not).
Should we add this twice? One running it on everything, the other only on the changes?

Reply:
I can update it to actually run it against all. The intent was just to give a way to run the existing pre-commit in the devcontainer since I cannot use the pre-commit hook (I use signed commits and commit signing does not work in the devcontainer).

Reply:
> The intent was just to give a way to run the existing pre-commit in the devcontainer since I cannot use the pre-commit hook (I use signed commits and commit signing does not work in the devcontainer).

GPG agent forwarding has been supported for quite some time already.

> I can update it to actually run it against all. 

Sounds like something one doesn't want. You might want to add it twice as suggested.

Reply:
> GPG agent forwarding has been supported for quite some time already.

Socket forwarding on Windows + WSL is a PITA. It supposed to work, but it does not. I also use a YubiKey for GPG as well, so it might be the smart card causing issues.

Reply:
Long-time Yubikey users here; GPG advocate.